banners/mobile/C24_WMDE_Mobile_DE_10/banner_var.ts

```diff
@@ -42,7 +44,8 @@ const app = createVueApp( BannerConductor, {
 		useOfFundsContent: localeFactory.getUseOfFundsLoader().getContent(),
 		pageScroller: new WindowPageScroller(),
 		remainingImpressions: impressionCount.getRemainingImpressions( page.getMaxBannerImpressions( 'mobile' ) ),
-		localCloseTracker: new LocalStorageCloseTracker()
+		localCloseTracker: new LocalStorageCloseTracker(),
+		currencyFormatter: currencyFormatter
```

Comment:
Please don't pass the currencyFormetter as a prop, it's injected with `provide` (see line 63) and can be accessed with `const currencyFormatter = inject('currencyFormatter')` in any component. We're trying to use props only for reactive values, not for "services" that do something (return a result, have side effects).